janitor/functions.py

```diff
@@ -189,8 +220,8 @@ def get_features_targets(df, target_columns, feature_columns=None):
     else:
         if isinstance(target_columns, str):
             xcols = [c for c in df.columns if target_columns != c]
-        elif (isinstance(target_columns, list)
-                or isinstance(target_columns, tuple)):
+        elif (isinstance(target_columns, list) or
+              isinstance(target_columns, tuple)):
```

Comment:
Logical and mathematical operators are put on a new line for clarity (particularly if there are multiple math operations going on). Please revert this.